blueocean-dashboard/src/main/js/components/records.jsx

```diff
@@ -70,7 +70,7 @@ export class RunRecord extends Record({
     _class: null,
     _capabilities: [],
     _links: null,
-    changeSet: ChangeSetRecord,
+    changeSet: [],
```

Comment:
`changeSet` is actually an array of these records. If JSON response omitted the prop, then it would get initialized to an empty `ChangeSetRecord` which would defeat null checking in the UI layer but still cause an exception to be thrown.